changelog/README.md

```diff
@@ -2,6 +2,9 @@
 
 ### 3.1.0 (in progress)
 
+- [new feature] JAVA-1157: Mapper should allow asynchronous Result paging.
```

Comment:
Minor: could you switch to an imperative style for this and the commit message? For example "Allow asynchronous Mapper Result paging".